pkg/codefresh/codefresh.go

```diff
@@ -36,6 +36,7 @@ type (
 	}
 
 	V2API interface {
+		UsersV2() UsersV2API
```

Comment:
does the field name need `V2`? can't it just be `Users`? it's already inside `V2API`
also - the interface should start with `I`, like the others

Reply:
I can't call it `Users` because there is already a `Users` function that is returning the V1 users api with that name